openedx/core/djangoapps/user_api/urls.py

```diff
@@ -26,16 +27,21 @@
     url(
         r'^v1/accounts/{}/image$'.format(settings.USERNAME_PATTERN),
         ProfileImageView.as_view(),
-        name="accounts_profile_image_api"
+        name='accounts_profile_image_api'
     ),
     url(
         r'^v1/preferences/{}$'.format(settings.USERNAME_PATTERN),
         PreferencesView.as_view(),
-        name="preferences_api"
+        name='preferences_api'
     ),
     url(
         r'^v1/preferences/{}/(?P<preference_key>[a-zA-Z0-9_]+)$'.format(settings.USERNAME_PATTERN),
         PreferencesDetailView.as_view(),
-        name="preferences_detail_api"
+        name='preferences_detail_api'
+    ),
+    url(
+        r'^v1/verification_status/{}$'.format(settings.USERNAME_PATTERN),
```

Comment:
Ugh! Someone screwed up by doing starting this weird pattern. Ideally, the URL should be `https://courses.edx.org/api/user/v1/accounts/{username}/verification_status/`.